src/redux/rocket/rocketSlice.js

```diff
@@ -0,0 +1,59 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const url = 'https://api.spacexdata.com/v4/rockets';
+
+export const getRockets = createAsyncThunk('rockets/fetch', async () => {
+  const response = await fetch(url);
+  if (!response.ok) {
+    throw new Error('Failed to fetch rockets');
+  }
+  const data = await response.json();
+  return data;
+});
```

Comment:
- [ ] The data from the API should be filtered based on [the requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/group-project/project_space_travelers_hub.md#redux-fetch-data-and-update-redux-store). Kindly filter the API data so that only the listed items in the requirements are saved to the store.

  ![image](https://github.com/harshi0102/space-hub/assets/36542986/c3b1bf05-b4ea-4091-ad45-288ca396445c)